packages/components/src/components/ContentfulContainer/ContentfulContainer.css

```diff
@@ -1,10 +1,10 @@
-#ContentfulContainer.defaultStyles {
+.ContentfulContainer {
   position: relative;
   display: flex;
   box-sizing: border-box;
 }
 
-#ContentfulContainer.defaultStyles::-webkit-scrollbar {
+.ContentfulContainer::-webkit-scrollbar {
```

Comment:
nitpick, but looks like we have a kebab casing going on for most css classes, so we might want to make this 'contentful-container'